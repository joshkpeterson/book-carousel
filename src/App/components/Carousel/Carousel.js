import "./Carousel.scss"
import React from "react"

export default function Carousel() {
  return <section className="carousel"></section>
}
