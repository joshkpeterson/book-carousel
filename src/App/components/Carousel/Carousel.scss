.carousel {
  .arrow {
    border: solid #57617a;
    border-width: 0 2px 2px 0;
    display: inline-block;
    padding: 5px;
    vertical-align: middle;
    transform: rotate(-45deg);
  }
}
