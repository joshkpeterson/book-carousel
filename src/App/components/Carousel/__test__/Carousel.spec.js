import Carousel from "../Carousel"
import React from "react"
import renderer from "react-test-renderer"

describe("<Carousel />", () => {
  it("matches the snapshot", () => {
    const component = renderer.create(<Carousel />)
    expect(component).toMatchSnapshot()
  })
})
