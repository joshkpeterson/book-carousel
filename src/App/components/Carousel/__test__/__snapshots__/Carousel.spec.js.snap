// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Carousel /> matches the snapshot 1`] = `
<section
  className="carousel"
/>
`;
